### ISSUE TITLE
Newest urllib3 breaks dxpy

### ISSUE THREAD

**Issue by jtratner** [dnanexus/dx-toolkit#746]
Would it be possible to guard the import so more of dxpy is usable for now (and then maybe figure out longer term fix later)
```  File "dxpy/__init__.py", line 151, in <module>
    from requests.packages.urllib3.packages.ssl_match_hostname import match_hostname
ModuleNotFoundError: No module named 'requests.packages.urllib3.packages.ssl_match_hostname'
```

**Comment by kpjensen** [comment]
Thanks for reporting, we will get out an updated release today as this import is no longer used in dxpy.

**Comment by kpjensen** [comment]
Released in v0.318.1
